code42_on_poll_connector.py

```diff
@@ -102,8 +102,18 @@ def __init__(self, connector, client, state):
 
     def handle_on_poll(self, param, action_result):
         param = self._adjust_date_parameters(param)
-        query = build_alerts_query(param["start_date"], param.get("end_date"))
+        severities = self._connector.get_config().get("severity")
+        if severities:
+            severities = severities.replace(" ", "").split(",")
+        query = build_alerts_query(
+            param["start_time"], param.get("end_time"), severities=severities
+        )
         alerts = self._get_alerts(param, query)
+        source_id = param.get("container_id")
```

Comment:
Are we sure that this parameter is meant to mean the source ID for the container (aka `source_data_identifier` when we init the container) rather than the event ID that is displayed in the UI?

Reply:
This is my understanding:
* The param `container_id` means "only get source events (Code42 alerts, in our case) with these Source IDs (Alert Ids) "
* The alerts are the containers, and the file event-observations of certain alerts become artifacts on the alert
* Artifacts are also a type of container, and both have the property `source_data_identifier` which just mean the external ID (the ID from our system instead of Phantom's).

Reply:
What's the use-case for polling for a single (or list of) alert ID(s)?

Reply:
this parameter exists in the UI _no matter what_... Some apps ignore it, but I figured since we can't make it go away, we might as well implement it.

A use case would be something like an analyst knows of a specific alert(s) that is in Code42 but for some reason it did not show up in Phantom.Now, they can manually ingest it.

Reply:
I just made it such that all other query parameters are ignored when specifying source IDs, which really makes the most sense, this was good feedback.

Reply:
gotcha, totally makes sense to implement something if the field is forced to be there either way.